### PR TITLE
chore: reprovision

### PR DESCRIPTION
**Pull request by danielmoll**
Hoping this will now publish!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -115,22 +116,22 @@
...
     "@economist/doc-pack": "^1.0.6",
-    "@economist/provision-react-component": "1.5.2",
+    "@economist/provision-react-component": "1.6.2",
     "babel": "^5.8.34",
     "babel-polyfill": "^6.6.1",
     "babelify": "^6.4.0",
     "browserify": "^13.0.0",
     "chai": "^3.5.0",
     "chai-spies": "^0.7.1",
-    "eslint": "~2.2.0",
```
Comment (gcedo) [inline package.json]: Pin `eslint` to `~2.2.0` or the build will fail as of an error in `2.3.0`

File: package.json [hunk 2]
```diff
@@ -115,22 +116,22 @@
...
+    "eslint": "^2.3.0",
     "eslint-config-strict": "^8.5.0",
     "eslint-config-strict-react": "^8.0.1",
     "eslint-plugin-filenames": "^0.2.0",
-    "eslint-plugin-react": "^4.2.1",
-    "ghooks": "^1.0.1",
+    "eslint-plugin-react": "^4.2.0",
```
Comment (gcedo) [inline package.json]: Downgrading here

File: package.json [hunk 3]
```diff
@@ -115,22 +116,22 @@
...
+    "ghooks": "^1.0.3",
     "git-directory-deploy": "^1.5.0",
     "hbs-cli": "^1.0.0",
-    "karma": "^0.13.22",
+    "karma": "^0.13.21",
```
Comment (gcedo) [inline package.json]: Downgrading here

File: package.json [hunk 4]
```diff
@@ -138,7 +139,7 @@
     "live-server": "^0.9.2",
     "mocha": "^2.4.5",
     "npm-assets": "^0.1.1",
-    "npm-run-all": "^1.5.2",
+    "npm-run-all": "^1.5.1",
```
Comment (gcedo) [inline package.json]: Downgrading here

File: package.json (hunk 2 above)
Comment (danielmoll) [inline package.json]: Do we not want to use the versions provision sets then?

**Comment by gcedo** [comment]
:rainbow:

**Pull request merged**